.circleci/config.yml

```
@@ -434,12 +434,17 @@ jobs:
             # - DOCS_VERSION: 1.1 (for git tag v1.1.2)
             # - DOCS_ALIAS: latest
             # or
-            # - DOCS_VERSION: master (for master branch)
+            # - DOCS_VERSION: main (for main branch)
             # - No DOCS_ALIAS
             DOCS_VERSION=$([[ -z "$CIRCLE_TAG" ]] && echo $CIRCLE_BRANCH || echo ${CIRCLE_TAG} | sed 's/^v\([0-9]\.[0-9]*\)\..*/\1/')
```

I find that a documentation version named `dev` less confusing than having a `main`. WDYT?
```suggestion
            # - DOCS_VERSION: dev (for main branch)
            # - No DOCS_ALIAS
            DOCS_VERSION=$([[ -z "$CIRCLE_TAG" ]] && echo "dev" || echo ${CIRCLE_TAG} | sed 's/^v\([0-9]\.[0-9]*\)\..*/\1/')
```

---

In this case, what about `development` instead?